src/RefractSourceMap.h

```diff
@@ -0,0 +1,73 @@
+//
+//  RefractSourceMap.h
+//  drafter
+//
+//  Created by Jiri Kratochvil on 31/07/15.
+//  Copyright (c) 2015 Apiary Inc. All rights reserved.
+//
+
+#ifndef DRAFTER_REFRACTSOURCEMAP_H
+#define DRAFTER_REFRACTSOURCEMAP_H
+
+#include "Serialize.h"
+
+namespace drafter {
+
+
+    refract::IElement* BytesRangeToRefract(const mdp::BytesRange& bytesRange);
+
+    template<typename T>
+    refract::IElement* SourceMapToRefract(const T& sourceMap) 
```

Comment:
I don't understand what's the difference between this and `SourceMapToRefract` from `RefractParseResult.h`. Why do we need `template` here? I think this function is unnecessarily more complex. Please move the other function here and use it.